src/lib/codeGeneration.ts

```
@@ -25,47 +44,57 @@ export function generateMCPCode(nodes: Node<BaseNodeData>[], mcpConfig: MCPConfi
-  let fixedArgs = [...mcpConfig.args];
-
-  // Update the MCP package in args if it's different from what's in smitheryMcp
-  if (mcpConfig.smitheryMcp) {
-    const runIndex = fixedArgs.indexOf('run');
-    if (runIndex !== -1 && runIndex + 1 < fixedArgs.length) {
-      // Replace the package after 'run' with the correct one
-      fixedArgs[runIndex + 1] = mcpConfig.smitheryMcp;
-    } else if (runIndex !== -1) {
-      // Add the package after 'run' if it doesn't exist
-      fixedArgs.splice(runIndex + 1, 0, mcpConfig.smitheryMcp);
+    let fixedArgs = [...cfg.args];
```

ESLint

[error] 'fixedArgs' is never reassigned. Use 'const' instead.

(prefer-const)